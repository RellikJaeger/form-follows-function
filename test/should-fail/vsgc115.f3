/*
 * Regression test: VSGC-115: misleading error message from f3c
 *
 * @test/compile-error
 *
 * This should produce:
 *
 *     vsgc115.f3:29: incompatible types
 *     found   : String
 *     required: Number
 *     var x:Number = new String("ss");
 *                    ^
 *     1 error
 *     
 * instead of
 *
 *     vsgc115.f3:29: incompatible types
 *     found   : java.lang.String
 *     required: double
 *     var x:Number = new String("ss");
 *                    ^
 *     1 error
 *      *
 * There doesn't seem to be a way to provide .EXPECTED files 
 * for tests that are expected to fail compilation.
 *
 */

var x:Number = new String("ss");
