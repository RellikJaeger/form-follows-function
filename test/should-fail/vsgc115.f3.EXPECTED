test/should-fail/vsgc115.f3:29: incompatible types
found   : String
required: Number
var x:Number = new String("ss");
               ^
1 error
